app/models/pipeline_run_stage.rb

```diff
@@ -41,6 +48,19 @@ def stage_status_file(status)
     "#{pipeline_run.sample.sample_output_s3_path}/#{basename}"
   end
 
+  def dag_json_name
+    STEP_NUMBER_TO_DAG_JSON_NAMES[step_number]
+  end
+
+  def step_status_file_path
+    path_beginning = if step_number <= 2
+                       pipeline_run.sample.sample_output_s3_path
+                     else
+                       pipeline_run.sample.sample_postprocess_s3_path
```

Comment:
What about experimental?

Reply:
Mentioned it below but it seems like all the experimental results are also being placed in the `postprocess` folder